src/main/scala/uk/gov/homeoffice/drt/repository/ArrivalRepository.scala

```diff
@@ -34,6 +38,7 @@ class ArrivalRepository[F[_] : Sync](val sessionPool: Resource[F, Session[F]]) e
         )
     }
 
+
   private def selectArrivalsForADate: Query[LocalDateTime ~ LocalDateTime, ArrivalTableData] =
     sql"""
         SELECT code , number , destination , origin , terminal , status ,scheduled ,scheduled_departure
```

Comment:
inconsistent formatting of the query string here - my preference is for `fieldName, fieldName2`